catalog/resource_grants.go

```diff
@@ -104,6 +104,7 @@ func (sm securableMapping) kv(d attributeGetter) (string, string) {
 		}
 		return field, v
 	}
+	fmt.Printf("[WARN] Unexpected resource or permissions. Please proceed at your own risk.")
```

Comment:
We should use `tflog.Warn` here rather than printing out information: https://pkg.go.dev/github.com/hashicorp/terraform-plugin-log/tflog#Warn

Reply:
Yes, sorry about that, didn't push the latest commits. I have also removed the map because we can use the one in permissions.